TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -161,6 +161,33 @@ TYPO3:
         'ru': 'Pусский – Russian'
 #        'sv': 'Svenska – Swedish'
 
+      # Aloha presets used in nodeType configuration
+      aloha:
+        presets:
+          'Standard':
```

Comment:
I'd suggest to use "standard" as preset name, as opposed to "Standard". In other places, we also use lowercase. Opinions?